engines/bops_api/app/controllers/bops_api/v2/planning_applications/neighbour_responses_controller.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+module BopsApi
+  module V2
+    module PlanningApplications
+      class NeighbourResponsesController < AuthenticatedController
+        def create
+          @planning_application = find_planning_application(params[:planning_application_id])
+          @consultation = @planning_application.consultation
```

Comment:
It's generally idomatic Rails controller style to move these to before actions, e.g.

``` ruby
before_action :set_planning_application
before_action :set_consultation
before_action :build_neighbour_response

def create
  @neighbour_response.save!

  respond_to do |format|
    format.json
  end
end

private

def set_planning_application
  @planning_application = find_planning_application(params[:planning_application_id])
end

def set_consultation
  @consultation = @planning_application.consultation
end

def build_neighbour_response
  @neighbour_response = @consultation.neighbour_responses.new(response_attributes)
end
```